node-fs-readfile/cat.js

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'node:fs/promises';
+
+const urlArray = process.argv.slice(2);
+
+async function readAll(arrayOfLinks) {
+  try {
+    const arrayOfPromises = arrayOfLinks.map((element) =>
+      fs.readFile(element, 'utf8')
+    );
+    const results = await Promise.all(arrayOfPromises);
+    results.map((element) => console.log(element));
+  } catch (error) {
+    console.log('ERROR: ', error);
+  }
+}
+
+if (urlArray.length > 0) {
+  readAll(urlArray);
```

Comment:
You could also `await` this. It will cause Node to pause executing the rest of the file until `readAll` completes, but you don't execute any code after this, so in this case it has no practical effect.